cli/azd/pkg/executil/executil.go

```diff
@@ -197,10 +198,31 @@ type RunArgs struct {
 	EnrichError bool
 }
 
+type RedactData struct {
+	matchString   *regexp.Regexp
+	replaceString string
+}
+
 func (rr RunResult) String() string {
 	return fmt.Sprintf("exit code: %d, stdout: %s, stderr: %s", rr.ExitCode, rr.Stdout, rr.Stderr)
 }
 
+func RedactSensitiveData(msg string) string {
+	regexpRedact := make(map[string]RedactData)
+	regexpRedact["access token"] = RedactData{
+		matchString:   regexp.MustCompile("\"accessToken\": \".*\""),
+		replaceString: "\"accessToken\": \"redact to prevent sensitive data\"",
+	}
+
+	for _, val := range regexpRedact {
+		regMatchString := val.matchString.FindString(msg)
+		if regMatchString != "" && strings.Contains(msg, regMatchString) {
+			return strings.Replace(msg, regMatchString, val.replaceString, -1)
```

Comment:
can we use [ReplaceAllString](https://pkg.go.dev/regexp#Regexp.ReplaceAllString) ??  I think we can avoid creating the `regMatchString`